src/formio/validators/minMaxDateAndDatetimeValidator.js

```diff
@@ -0,0 +1,43 @@
+import set from 'lodash/set';
+
+import {validateBoundaries} from './utils';
+
+const createValidator = key => {
+  return {
+    key,
+    message(component) {
+      const minDate = new Date(component.component.minDate);
+      const maxDate = new Date(component.component.maxDate);
```

Comment:
shouldn't this also use `parseISO` for dates, otherwise we end up with the same timezone issues again?

Reply:
These are only used for the error message formatting, but I think switching to `parseISO` is fine. I also see we are using different attributes from the `check` call, I'll look into that.

Reply:
I'd rather have `parseISO` be applied consistently, otherwise next time someone will be tempted to use `new Date` on a date (rather than datetime) string again and then we have the same timezone bug again.